### ISSUE TITLE
If renamedEntry==file the File is Deleted

### ISSUE THREAD

**Issue by emmercm** [alex-ketch/metalsmith-renamer#4]
I'm attempting to use this plugin like:

```js
    // Lowercase extensions
    .use(renamer({
        lowercase: {
            pattern: '**/*.*',
            rename: file => {
                const split = file.split('.');
                return split.slice(0, split.length - 1).join('.') + '.' + split[split.length-1].toLowerCase();
            }
        }
    }))
```

But because of the line in the plugin:

```js
        files[renamedEntry] = files[file];
        delete files[file];
```

Any filenames that don't change end up getting removed from the file list.

I'd expect them to not be removed.

**Comment by alex-ketch** [comment]
Hi @emmercm, thanks for reporting this!
I actually haven't used metalsmith in a while, and can't make promises of when I'll get to working on this as it will take me a bit to refresh my memory.
Would you feel comfortable in making a PR?

If I understood you correctly, the problem arises when `renamedEntry === file`, is that right?

**Comment by emmercm** [comment]
I can definitely make the PR, though it might take me a couple extra days.

That's right - the thought is I'd only `delete files[file]` if `file !== renamedEntry`.

**Comment by alex-ketch** [comment]
Closed by #5
